src/views/List.jsx

```diff
@@ -22,7 +26,22 @@ export function List({ data }) {
 		setFilterInput('');
 	};
 
-	return (
+	const handleAddItem = () => {
+		/* navigate to add item view */
+		navigate('/add-item');
+	};
+
+	return data.length == 1 ? (
```

Comment:
Good to get in the habit of using strict equality checks

```suggestion
	return data.length === 1 ? (
```

Reply:
@djtaylor8 I think this was the other pair's code but I believe they've changed this to a `map` instead? (from what I remember seeing during the demo). I wonder then if this can cause a merge conflict..?

Reply:
Ah that's weird--not sure how that got into our code? I just went through and deleted the lines that somehow got migrated in from the branch the other group was working on.

Reply:
That can sometimes happen if you have uncommitted changes on your current branch. All good though!